wizeline/wize-q-remix
# feat: add feature flagging for employee selected requirement

#### What does this PR do?
- Adds feature flag json to activate/deactivate easily future features.

#### Where should the reviewer start?
- Toggle this flag as desired for testing (preferably we should test both scenarios)

#### How should this be manually tested?
1. Go to question create page
2. If flag is false, then you should be able to create a question without seeing the dropdown for the employee
3. On the contrary, you should see the employee dropdown and validation that you cannot create the question until selecting one.

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #201
